microsoft-edge/webview2/concepts/apis-capabilities.md

```diff
@@ -0,0 +1,1422 @@
+---
+title: Overview of WebView2 APIs and their capabilities
+description: Overview of WebView2 APIs and their capabilities.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/20/2022
+---
+# Overview of WebView2 APIs and their capabilities
+
+<!-- todo:
+*  pull in/merge in the Overview/Hosting doc
+*  condense overview of top 3 classes down to 1 paragraph copied to detailed page
+-->
+
+This article provides:
+*  A high-level understanding of the capabilities of the WebView2 technology.
+*  Concepts about what you can do with the WebView2 APIs.
+*  High-level description of WebView2 technology for app developers.
+*  Better fundamentals + understanding of WebView2 technology.
+
+Audience: App developers who are curious about the WebView2 technology and would like to know if WebView2 is for them.
+
+Top-level summary of APIs:
+
+| API | Purpose |
```

Comment:
@nishitha-burman - compare other usage of overview tables:
* https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/about-tools#overview-of-all-tools
* https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/utilities#summary